rust/chains/hyperlane-sealevel/src/interchain_gas.rs

```diff
@@ -74,3 +76,11 @@ impl Indexer<InterchainGasPayment> for SealevelInterchainGasPaymasterIndexer {
         Ok(1)
     }
 }
+
+#[async_trait]
+impl SequenceIndexer<InterchainGasPayment> for SealevelInterchainGasPaymasterIndexer {
+    async fn nonce_at_tip(&self) -> ChainResult<(u32, u32)> {
```

Comment:
nit: nonce -> sequence for consistency

Reply:
have renamed everywhere except for the actual functions that use a nonce, e.g. `retrieve_message_by_nonce`, let me know if that makes sense